### PR TITLE
namespace debian package with library version

### PR DESCRIPTION
**Pull request by s3rj1k**
Signed-off-by: s3rj1k <evasive.gyron@gmail.com>

### CODE REVIEW

File: src/Makefile.am [hunk 1]
```diff
@@ -84,7 +84,7 @@ EXTRA_DIST = floating_fudge.h \
 
 AM_CPPFLAGS = -I$(top_builddir)
 
-lib_LTLIBRARIES = libspandsp.la
+lib_LTLIBRARIES = libspandsp3.la
```
Comment (mjerris) [inline src/Makefile.am:87]: we don't want to change this version, just the version of the so

File: debian/rules [hunk 2]
```diff
@@ -2,25 +2,30 @@
...
 	dh $@ --with autoreconf
 
-INC_DIR=$(CURDIR)/debian/tmp/usr/include
-INC_DIR_ARCH=$(INC_DIR)/$(DEB_BUILD_MULTIARCH)
+INC_DIR=$(CURDIR)/debian/tmp/usr/include/spandsp3/
+INC_DIR_ARCH=$(INC_DIR)/$(DEB_BUILD_MULTIARCH)/
 
 override_dh_auto_configure:
-	dh_auto_configure -- --enable-doc --with-pic
+	dh_auto_configure -- --enable-doc --with-pic --prefix=/usr --includedir=/usr/include/spandsp3/ --libdir=/usr/lib/$(DEB_BUILD_MULTIARCH)/spandsp3/
```
Comment (mjerris) [inline debian/rules:9]: we dont want to change libdir

File: debian/libspandsp3.install [hunk 3]
```diff
@@ -0,0 +1 @@
+debian/tmp/usr/lib/*/spandsp3/libspandsp3.so.*
```
Comment (mjerris) [inline debian/libspandsp3.install:1]: we should be doing libspandsp.so.X where x is bigger than what the other packages are doing

File: debian/libspandsp3-dev.install [hunk 4]
```diff
@@ -0,0 +1,5 @@
+debian/tmp/usr/include/spandsp3/*/spandsp.h
+debian/tmp/usr/include/spandsp3/spandsp
+debian/tmp/usr/lib/*/spandsp3/libspandsp3.so
```
Comment (mjerris) [inline debian/libspandsp3-dev.install:3]: we shouldn't be stuffing libs in a different dir

File: debian/libspandsp3.install (hunk 3 above)
Comment (s3rj1k) [reply debian/libspandsp3.install:1]: If we do not change library name, pkg-config file would be named as the one that stock spandsp package bring. (spandsp.pc)

If we do manually rename spandsp.pc to spandsp3.pc, there would be still issue with pkg-config looking-up for proper library. (`Libs: -L${libdir} -lspandsp` vs `Libs: -L${libdir} -lspandsp3`)

Only way that I can see to solve this, is to set major version to some X, make library package coinstallable with upstream package, make devel package replace upstream package so pkg-config would work.

File: debian/libspandsp3-dev.install (hunk 4 above)
Comment (s3rj1k) [reply debian/libspandsp3-dev.install:3]: this is related to `we dont want to change libdir`, can be removed when we solve pkg-config issue

File: debian/rules (hunk 2 above)
Comment (s3rj1k) [reply debian/rules:9]: depends on pkg-config issue

**Pull request closed without merging**